cpp/src/decisiontree/batched-levelalgo/kernels.cuh

```diff
@@ -35,6 +35,7 @@ namespace DecisionTree {
 template <typename IdxT>
 struct WorkloadInfo {
   IdxT nodeid;  // Node in the batch on which the threadblock needs to work
+  IdxT large_nodeid;    // counts only large nodes
```

Comment:
Make sure you comment what large nodes means.

Reply:
done 👍🏾